src/docs/organizations.yaml

```diff
@@ -0,0 +1,691 @@
+tags:
+    - name: Organization
+      description: API to manage organization profile.
+paths:
+    /api/organization/:id:
+        get:
+            tags:
+                - Organization
+            summary: View organization public profile
+            description: show public information of an organization
+            operationId: getOrganization
+            consumes:
+                - multipart/form-data
+            produces:
+                - application/json
+            parameters:
+                - name: id
+                  in: path
+                  required: true
+                  schema:
+                      type: string
+                      example: 5e9f8f8f8f8f8f8f8f8f8f8
+            responses:
+                200:
+                    description: The organization Profile.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/FullPublicOrganization'
+                404:
+                    description: Organization not found  or has an invalid value
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ValidationError'
```

Comment:
You can't use $ref and properties at the same time. If you look at your endpoint in swagger, you will see that only the $ref is visible, and non of the stuff in the properties is visible. 

Also, you are referencing the ValidationError, which has a different schema than yours. You have `{message: 'Page not found'}`.

So, you should add both your 404 error and the 422, which is the validation error.